modules/op-adding-resources-in-database-for-tekton-hub.adoc

```diff
@@ -0,0 +1,48 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-tekton-hub-with-openshift-pipelines.adoc
+
+:_content-type: PROCEDURE
+[id="adding-resources-in-the-database-for-tekton-hub_{context}"]
+= Adding resources in the database for {tekton-hub}
+
+[discrete]
+.Prerequisites
+* Ensure that you are in the `<project_root>/config/` directory, where `<project_root>` is the top level directory of the cloned {tekton-hub} repository.
+
+[discrete]
+.Procedure
+. In the {tekton-hub} UI, click **Login** --> **Sign in with GitHub/Github Enterprise**.
+
+. On the home page, click the user profile and copy the token.
+
+. Call the Catalog Refresh API.
+* To refresh a catalog with specific name, run the following command:
++
+[source,terminal]
+----
+$ curl -X POST -H "Authorization: <jwt-token>" \ <1>
+  <api-url>/catalog/<catalog_name>/refresh <2>
+----
+<1> Replace the authorization token with the {tekton-hub} token copied from UI
+<2> Replace the `<api-url>` with the API pod URL, and `catalog_name` with the name of the catalog.
```

Comment:
You can shorten this instance, and other similar instances, by removing the "Replace the..." part and simply naming what the replacement value is.
```suggestion
<1> The {tekton-hub} token copied from UI
<2> The API pod URL and name of the catalog.
```

Reply:
If you do a global search for `<1>` you can see how this is typically done in the OpenShift docs.